packages/grid/src/elements/splitter/Pane.tsx

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useState, useMemo, HTMLAttributes, forwardRef } from 'react';
+import { Splitter } from './components/Splitter';
+import { Content } from './components/Content';
+import { StyledPane } from '../../styled';
+import { PaneContext } from '../../utils/usePaneContext';
+
+const PaneComponent = forwardRef<HTMLDivElement, HTMLAttributes<HTMLDivElement>>(
+  ({ children, ...props }, ref) => {
+    const [paneId, setPaneId] = useState<string>();
+
+    const paneContext = useMemo(
+      () => ({
+        id: paneId,
+        setId: (id: string) => setPaneId(id)
+      }),
+      [paneId, setPaneId]
```

Comment:
```suggestion
      [paneId]
```
Do we need `setPaneId` in the dependencies array?

Reply:
Fixed in [7df0ecb](https://github.com/zendeskgarden/react-components/pull/1339/commits/7df0ecb7cc11861806fb2838667fa5947e7b80b2)